rpm will fail pkgconf test pipeline

If the pkgconf test pipeline is added to rpm it will fail in the following way:

```
2024/11/12 23:15:46 INFO running step "test/pkgconf"
2024/11/12 23:15:46 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:15:46 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:15:46 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:15:46 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:15:46 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN + basename /home/build/melange-out/rpm-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN + dev_pkg=rpm-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN + apk info -L rpm-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN + grep -q ^Name: usr/lib/pkgconfig/rpm.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN + basename usr/lib/pkgconfig/rpm.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN + lib_name=rpm uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN + echo usr/lib/pkgconfig/rpm.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN + grep -q '^usr/lib/pkgconfig/rpm.pc$\|^usr/share/pkgconfig/rpm.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN + pkgconf --exists rpm uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN + pkgconf --print-errors --exists rpm uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN Package popt was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN Perhaps you should add the directory containing `popt.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 WARN Package 'popt', required by 'rpm', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:15:46 INFO pod 7c89c45bf35ce3a9be291c21d66d0e0e8177ad5b560ef238787095b6751a8236 terminated
2024/11/12 23:15:46 INFO pod 2dbd54fccb3466807d59710d63ad94b6b256cddb43ddbcdd7455914e8ccd7308 terminated
2024/11/12 23:15:46 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:15:46 INFO   workspace dir: /home/user/tmp/melange-workspace-513731817
2024/11/12 23:15:46 INFO   guest dir: /home/user/tmp/melange-guest-4023410052
2024/11/12 23:15:46 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/rpm] Error 1
```

--- a/rpm.yaml
+++ b/rpm.yaml
@@ -2,7 +2,7 @@
 package:
   name: rpm
   version: 4.19.1
-  epoch: 1
+  epoch: 2
   description: Redhat Package Management utilities (RPM)
   copyright:
     - license: GPL-2.0-or-later OR LGPL-2.0-or-later
@@ -81,6 +81,12 @@ subpackages:
     pipeline:
       - uses: split/dev
     description: rpm dev
+    dependencies:
+      runtime:
+        - popt-dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: rpm-doc
     pipeline:

